pitch.md

```diff
@@ -23,38 +86,122 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
+
+It should be difficult or almost impossible to produce a complete song with
+this language. The language is not a DAW; that is, it is not meant to be used
+for recording, mixing or mastering. More likely, the user would need to combine
+instruments written in this language with a fully-feature DAW to produce a
+complete song.
 
 ## Related work
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well 
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+CSound and any DAW would fall into this domain. See above for a description of
+these tools, their advantages, and their problems.
 
 ## Suitability
 _What percentage of your time do you think will be spent directly engaging in
 the **language** aspects of this project (e.g., making language design
 decisions), as opposed to "systems" aspects of the project (e.g., implementing a
 complicated semantics that doesn't require a lot of language design)?_
 
+I plan on leveraging CSound to implement the semantics of my language. CSound
+comes with tools for generating audio from instruments, and for interfacing
+with MIDI data. This means that the only semantics I would have to implement
+would be the translation from CSound++ to CSound, which should be quite
+feasible. This will leave lots of time for language decisions.
 
 ## Scope
 _How big an idea is this? How ambitious is this project?_
 
+I think this project is distinctly doable, given that most of the technical work could leverage existing solutions as described above. In fact, I believe it would be possible (though probably not recommended) to implement a prototype of this language using the CSound preprocessor.
 
 ## Challenges and opportunities
 _Why is this a good project? What are some challenges you expect to face? How
 might you overcome them? What are some warning signs that the project has gotten
 off track, and how will you get the project back on track if needed?_
+
+This project should emphasize the extensibility of the language, as advocated
+for by Steele, since it focuses on a small set of primitives which can be
+combined to create new primitive-like objects. This focus on composition and
+extensibility should make for a good case study in language design.
+
+I expect to meet a significant challenge in deciding what features to add. I
+have a tendency to get excited and run from one feature to the next without
+really finishing any of them, especially in a project like this where the
+domain interests me -- I can already think of way more features I would like to
+implement than I will probably have time for. Overcoming this challenge will
+require foresight, planning, and an honest evaluation of which features are
+necessary for a minimum viable project.
```

Comment:
Great that your interest in it provides a challenge :). I do think that being focused and outlining the most important features is important for this domain since there are so many features you could implement for a DSL related to music.